README.md

```diff
@@ -126,14 +126,16 @@ The following options are supported in `.tf_wrapper`:
 
 ```yaml
 configure_backend: True # If true, automatically configure Terraform backends.
-pipeline_check: True # If true, require this directory to be in a pipeline file.
```

Comment:
Was this missed from an earlier change?

Reply:
It's still supported but we've removed all references to it from the rest of the README so thought it should be removed from here too so its clear that it's deprecated